license-template.kt

```diff
@@ -0,0 +1,3 @@
+// SPDX-FileCopyrightText: Contributors to the GXF project
```

Comment:
Shouldn't this become:
```suggestion
// SPDX-FileCopyrightText: Copyright Contributors to the GXF project
```
?

Reply:
fixed